topics/proteomics/tutorials/metaquantome-data-creation/workflows/main_workflow.ga

```diff
@@ -11,15 +11,15 @@
             "id": 0,
             "input_connections": {},
             "inputs": [],
-            "label": null,
-            "name": "Input dataset collection",
+            "label": "Input_collection",
+            "name": "Input_collection",
             "outputs": [],
             "position": {
                 "left": 267.953125,
                 "top": 714.65625
             },
             "tool_id": null,
-            "tool_state": "{\"optional\": false, \"collection_type\": \"list\"}",
+            "tool_state": "{\"name\": \"good_outcome\"}",
```

Comment:
this looks wired, do you know why this is needed?

Reply:
This is a copy/paste mistake from the non-collections workflow test example. Fixing now.